designs/manager_policy_mapping/policy_mapping.md

```diff
@@ -26,7 +26,7 @@ The appliance security manager will not delete a security group with one or more
     "virtualSystemId": 0,
     "name": "string",
     "policyId": 0,
-    "managerSecurityGroupId": "string", // This new property is being added to store the security group id returned by the manager
+    "managerSecurityGroupId": "string", // This new property is added to persist the manager security group id
     "failurePolicyType": "FAIL_OPEN",
     "order": 0,
     "policies": [
```

Comment:
add comment on how this is now a list.

Also add information on what API is impacted with this change(with path and whether its a PUT/GET/POST etc